syntax_and_semantics/annotations.md

```diff
@@ -78,6 +78,39 @@ end
 annotation_value # => 2
 ```
 
+The `named_args` method can be used to read all key/value pairs on an annotation as a `NamedTupleLiteral`.
+
+```crystal
+annotation MyAnnotation
+end
+
+@[MyAnnotation(value: 2, name: "Jim")]
+def annotation_named_args
+  {{@def.annotation(MyAnnotation).named_args}}
```

Comment:
I think the formatter would go like this:
```suggestion
  {{ @def.annotation(MyAnnotation).named_args }}
```

Reply:
Formatter doesn't currently do anything to it.

Reply:
It probably should... I'd prefer it that way and equal to all `{%` macro tags.

Reply:
Indeed.  Would be nice.